src/webgl/light.js

```diff
@@ -284,4 +284,34 @@ p5.prototype.pointLight = function(v1, v2, v3, x, y, z) {
   return this;
 };
 
+/**
+ * Sets a default value for ambient and directional light
+ * @method lights
+ * @chainable
+ * @example
+ * <div>
+ * <code>
+ * function setup() {
+ *   createCanvas(100, 100, WEBGL);
+ * }
+ * function draw() {
+ *   background(0);
+ *   lights();
+ *   rotateX(millis() / 1000);
+ *   rotateY(millis() / 1000);
+ *   rotateZ(millis() / 1000);
+ *   box();
+ * }
+ * </code>
+ * </div>
+ *
+ * @alt
+ * light is different depending on the position of the object
```

Comment:
I've changed it into _'the light is partially ambient and partially directional'_, since the other alt lines I saw in the file were referring to the example. That's the best thing I came up with, is it good enough?

Reply:
Should be good thanks!